.github/workflows/ci.yaml

```diff
@@ -20,6 +20,11 @@ jobs:
           key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
           restore-keys: |
             ${{ runner.os }}-go-
+      - uses: arduino/setup-protoc@master
+      - name: Generete proto
+        run: |
+          go get -u github.com/golang/protobuf/protoc-gen-go
+          protoc --proto_path=./proto --go_out=plugins=grpc:./proto/go/degitxpb --go_opt=paths=source_relative ./proto/*.proto
```

Comment:
@stepanvalyavskiy can we use some `Makefile` with `protoc` cmd to make a convenient way of generating proto sources on CI builds and local machine both? Also, we can mention it in the readme file